CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Remove the type definitions deprecated in 2.0.0
 [Will McGinty](https://github.com/wmcginty)
 [#72](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/72)
+* Added failing type information to `DecodingFailureInitializable` allowing the API to make decisions based off of the type that failed to decode and deprecate dynamically keyed decoding.
```

Comment:
Minor, but can we get a newline before this entry?

Reply:
Fixed. Stupid merge.